browser/templates/browser/person.html

```diff
@@ -2,12 +2,47 @@
 {% load app_filters %}
 {% block title %}{{ person.name }} (Person){% endblock %}
 
-
+{% load staticfiles %}
 {% block main %}
-
 <script>
         $(document).ready(function() {
+
+            var person_url_params = window.location.href.split("?tab=");
+            if(typeof person_url_params[1] !== 'undefined' && typeof person_url_params[1] !== 'null' )
+            {
+                $('.nav-tabs a[href="#' + person_url_params[1] + '"]').tab('show');
+            }
```

Comment:
what about my previous comment here?

Reply:
Sorry about that. I am now using Django's request parameter to get information of tab parameter as following.
```
var tab = "{{request.GET.tab}}";
            if(typeof tab !== 'undefined' && typeof tab !== 'null' )
            {
                $('.nav-tabs a[href="#' + tab + '"]').tab('show');
            }
```

Reply:
just use django template 'if' and get rid of the ugly javascript.

Reply:
Changed to Django if condition as following:
```
{% if request.GET.tab %}
$('.nav-tabs a[href="#' + "{{request.GET.tab}}" + '"]').tab('show');
{% endif %}
```